account_avatax/views/account_move_view.xml

```diff
@@ -14,6 +14,9 @@
                     attrs="{'invisible': [('state', 'not in', ['draft'])]}"
                 />
             </button>
+            <xpath expr="//button[@name='action_reverse']" position="attributes">
+                <attribute name="context">{'avatax_amt_line_override': True}</attribute>
+            </xpath>
```

Comment:
Why is this? We should be able to reverse invoices without using Avatax override.